optimize/pom.xml

```diff
@@ -72,6 +72,9 @@
     <dependencyAnalyzeFailOnWarning>false</dependencyAnalyzeFailOnWarning>
     <dependencyAnalyze.failOnWarning>${dependencyAnalyzeFailOnWarning}</dependencyAnalyze.failOnWarning>
 
+    <!-- OpenCsv-->
+    <opencsv.version>5.10</opencsv.version>
```

Comment:
This is better but it should go in `parent/pom.xml`.
I think the convention in there is `version.opencsv` insteaf of `opencsv.version`